Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/VisualThemes/Core/ThemeDefinition.cs

```diff
@@ -136,5 +136,33 @@ public Easing Easing
 
             return null;
         }
+
+        void ISerializationCallbackReceiver.OnBeforeSerialize()
```

Comment:
add public specifier

Reply:
_The public keyword is not allowed on an explicit interface declaration_